### DISCUSSION TITLE
Radiomaster TX16S Companion "InternalAccess"

### DISCUSSION THREAD

**Discussion by kochpet** [EdgeTX/edgetx#3180] [Q&A] [answered]
In the companion there is no selection point for InternalAccess for the Radiomaster TX16S.

Hi, I converted my Radiomaster TX16S MKII to internal ACCESS. Can you please add the firmware selection point "InternalAccess" for the TX16S transmitter profile in the Companion.

**Comment by raphaelcoeffic** [comment, marked as answer]
You can use your new ISRM with the regular firmware by just selecting your internal module.

In the radio UI it is located in the Radio setting / Hardware:
<img width="350" alt="Screenshot 2023-02-11 at 15 18 18" src="https://user-images.githubusercontent.com/1050031/218262979-88e92f10-ae6f-4b16-808b-70995d03d826.png">

In Companion you can select it in the general radio profile:
<img width="704" alt="Screenshot 2023-02-11 at 15 22 17" src="https://user-images.githubusercontent.com/1050031/218263050-014a098d-7716-4576-b411-a80143771c27.png">

And in the file specific radio settings:
<img width="487" alt="Screenshot 2023-02-11 at 15 23 19" src="https://user-images.githubusercontent.com/1050031/218263089-e0fe0ebb-4c4a-40a9-9818-3c74c20c5deb.png">

Please note that when you create a new file in Companion, the internal module will be set to what is set in the profile setting.

You will also find this radio setting described in our user manual here:
https://edgetx.gitbook.io/edgetx-user-manual/edgetx-user-manual/user-manual-for-color-screen-radios/radio-settings/hardware

**Comment by 3djc** [comment]
Yeap, EdgeTX does not use InternalAccess, simply define your internal module as ISRM
